### ISSUE TITLE
External: Reflect Protocol in Social Web Protocols document

### ISSUE THREAD

**Issue by azaroth42** [w3c/web-annotation#118]
@azaroth42 @rhiaro to ensure that Social Web Protocols document has entries for where the Annotation Protocol has similar features to others in the space.

Document is:
  https://github.com/w3c-social/social-web-protocols/blob/gh-pages/social-web-protocols.md
(Update with github.io page when GH refreshes)

**Comment by azaroth42** [comment]
Blocked by issues described in the SWWG equivalent ticket: https://github.com/w3c-social/social-web-protocols/issues/18

**Comment by azaroth42** [comment]
Propose close wontfix. If SWWG want to reference us, they can.

**Comment by rhiaro** [comment]
Fine by me
